### PR TITLE
Improved logging.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java [hunk 1]
```diff
@@ -34,6 +39,7 @@ public void save(String sagaId, Set<EnlistedAggregate> enlistedAggregates) {
               try {
                 return new EnlistedAggregate(ClassUtils.getClass(rs.getString("aggregate_type")), rs.getString("aggregate_id"));
               } catch (ClassNotFoundException e) {
+                logger.error("Class not found", e);
                 throw new RuntimeException();
```
Comment (cer) [inline eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: `new Runtime...("message", e)`

Comment (dartartem) [reply eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: Fixed

File: eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public void save(String sagaId, Set<EnlistedAggregate> enlistedAggregates) {
                 return new EnlistedAggregate(ClassUtils.getClass(rs.getString("aggregate_type")), rs.getString("aggregate_id"));
               } catch (ClassNotFoundException e) {
                 logger.error("Class not found", e);
-                throw new RuntimeException();
+                throw new RuntimeException("Class not found", e);
```
Comment (cer) [inline eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: This message simply repeats the exception and so provides no useful information.
A more useful message would be one that describes *what* class isn't found, e.g. `RuntimeException("Aggregate type  not found", e)`

Comment (dartartem) [reply eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: ok, message for exception is "Aggregate type not found", should I use also logging here?

Comment (cer) [reply eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: Isn't that on line 42?

Comment (dartartem) [reply eventuate-tram-sagas-orchestration/src/main/java/io/eventuate/tram/sagas/orchestration/EnlistedAggregatesDao.java]: Yes, should I keep it as is?

**Pull request merged**